<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1411</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KStarsDataEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Stars</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>64</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>120</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Named Stars Header</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <widget class="QSpinBox" name="namedStarsVersion">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>30</y>
             <width>61</width>
             <height>27</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>101</width>
             <height>26</height>
            </rect>
           </property>
           <property name="text">
            <string>Version:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>101</width>
             <height>26</height>
            </rect>
           </property>
           <property name="text">
            <string>Description:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="namedStarsDescription">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>0</y>
             <width>501</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="namedStarsDataElement"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Named Stars Entry</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="namedStarsEntry"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>26</height>
       </size>
      </property>
      <widget class="QPushButton" name="loadButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>80</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>0</y>
         <width>80</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QPushButton" name="defaultButton">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>80</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1411</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
